sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeHealthcareEntitiesOperation.cs

```diff
@@ -258,15 +258,15 @@ public virtual async Task CancelAsync(CancellationToken cancellationToken = defa
         /// <remarks>
         /// Operation must complete successfully (HasValue is true) for it to provide values.
         /// </remarks>
-        public override AsyncPageable<AnalyzeHealthcareEntitiesResultCollection> GetValuesAsync() => _operationInternal.Value;
+        public override AsyncPageable<AnalyzeHealthcareEntitiesResultCollection> GetValuesAsync(CancellationToken cancellationToken = default) => _operationInternal.Value;
```

Comment:
We might need to call `CreateOperationValueAsync` here and use the provided token.

Reply:
ahh thank you. I was confused here